### ISSUE TITLE
Implement TF profiler on the code

### ISSUE THREAD

**Issue by umang-sh** [thisisashukla/inclusive-ai#3]
Implement TF profiler on the code.
https://www.tensorflow.org/guide/data_performance_analysis

Add callbacks to the code and make sure profiler is working.

Acceptance criteria:
- [ ] We should be able to see the profiling and operations on the profiler of inclusive AI code

**Comment by Jagadeesh1111** [comment]
Was able to get the scalars, need to look more on visulization

**Comment by Jagadeesh1111** [comment]
We are being able to launch it on AWS using tf2.0

**Comment by dp1411** [comment]
Test the tensorboard for more epochs

**Comment by dp1411** [comment]
Temporarily parked due to the issue of pytorch, tensorflow compatibility

**Comment by thisisashukla** [comment]
> Temporarily parked due to the issue of pytorch, tensorflow compatibility

why is there a pytorch issue? are we using both pytorch and tf?

**Comment by dp1411** [comment]
Yes we were using torch tensorboard as we were facing issues with tf tensorboard
